homeworks/02-image-layout-draft1/README.md

```diff
@@ -0,0 +1,9 @@
+This is first draft, actual solution is located in `02-image-layout` folder.
+
+Homework to make image layout.
+
+# Demo
+TODO
```

Comment:
Please, deploy and provide a link to the demo app :)

Reply:
Done, so there are 2 demos for this homework now.